Robinhood/Robinhood.py

```diff
@@ -334,15 +341,10 @@ def get_historical_quotes(self, stock, interval, span, bounds=Bounds.REGULAR):
         if isinstance(bounds, str):  # recast to Enum
             bounds = Bounds(bounds)
 
-        params = {
-            'symbols': ','.join(stock).upper(),
-            'interval': interval,
-            'span': span,
-            'bounds': bounds.name.lower()
-        }
+        historicals = endpoints.historicals() + "/?symbols=" + ','.join(stock).upper() + "&interval=" + interval + "&span=" + span + "&bounds=" + bounds.name.lower()
 
-        res = self.session.get(endpoints.historicals(), params=params, timeout=15)
-        return res.json()
+        res = self.session.get(historicals, timeout=15)
+        return res.json()['results'][0]
```

Comment:
Looking into this..

Reply:
You're right. I changed this back. I took the code off @anthonykrivonos . I wasn't sure why it was changed to that way and I had to change my bot's code because of it. Good call.